x/docs/internal/types/msgs.go

```diff
@@ -34,87 +40,56 @@ func (msg MsgShareDocument) Route() string { return ModuleName }
 // Type Implements Msg.
 func (msg MsgShareDocument) Type() string { return MsgTypeShareDocument }
 
-func validateUuid(uuid string) bool {
-	regex := regexp.MustCompile(`[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}`)
-	return regex.MatchString(uuid)
-}
-
-func validateDocMetadata(docMetadata DocumentMetadata) sdk.Error {
-	if len(docMetadata.ContentUri) == 0 {
-		return sdk.ErrUnknownRequest("MetadataSchema content URI can't be empty")
+// ValidateBasic Implements Msg.
+// TODO: Test more
+func (msg MsgShareDocument) ValidateBasic() sdk.Error {
+	if msg.Sender.Empty() {
+		return sdk.ErrInvalidAddress(msg.Sender.String())
 	}
 
-	if (docMetadata.Schema == nil) && len(docMetadata.SchemaType) == 0 {
-		return sdk.ErrUnknownRequest("Either schema or schema_type must be defined")
+	if msg.Recipients.Empty() {
+		return sdk.ErrUnknownRequest("Recipients cannot be empty")
```

Comment:
This error probably should be an `sdk.ErrInvalidAddress` like the one above

Reply:
Good point, changed this